CMakeLists.txt

```diff
@@ -432,12 +432,13 @@ if(BUILD_TESTS)
     )
 
     add_unit_test(
-      node_frontend_test
+      node_frontend_test ${CMAKE_CURRENT_SOURCE_DIR}/src/js/wrap.cpp
```

Comment:
Probably fine, but not obvious to me why this change is needed.

Reply:
It's because `node_frontend_test.cpp` now includes `frontend_test_infra.h` -> `member_frontend.h` -> `wrap.h`. We could build this separately if we refactor our libraries, or break this dependency chain (I don't know if the common test stuff actually _needs_ the member frontend's JS support?) Including this to match similar other unit tests.